### ISSUE TITLE
Bug: If geometric leaf clipping is deactivated, the value for delta_v is wrong

### ISSUE THREAD

**Issue by ypotdevin** [ypotdevin/dpgbdt#7]
Relevant section in the C++-Code:
```cpp
// sensitivity for leaves
if (params->gradient_filtering && !params->leaf_clipping) {
    // you can only "turn off" leaf clipping if GDF is enabled!
    tree_params.delta_v = params->l2_threshold / (1 + params->l2_lambda);
} else {
    tree_params.delta_v = std::min((double) (params->l2_threshold / (1 + params->l2_lambda)),
            2 * params->l2_threshold * pow(1-params->learning_rate, tree_index));
}
```

**Comment by ypotdevin** [comment]
Presumably, this Python code section from `model.py` has to be changed:
```python
def Train(self,
            X: np.array,
            y: np.array) -> 'GradientBoostingEnsemble':
    """Train the ensembles of gradient boosted trees.

    …
    """

    …

    # Train all trees
    for tree_index in range(self.nb_trees):
      # Compute sensitivity
      delta_g = 3 * np.square(self.l2_threshold)
      delta_v = min(self.l2_threshold / (1 + self.l2_lambda),
                    2 * self.l2_threshold * math.pow(
                      (1 - self.learning_rate), tree_index))
      …
